src/components/RecipeView.js

```diff
@@ -1,21 +1,37 @@
 import React from 'react';
 import Pagination from './Pagination';
 import Recipe from './Recipe';
+import '../styles/RecipeView.css';
 
 export default function RecipeView(props) {
-  const { onPageChange, data, currentPage, totalCount, pageSize } = props;
+  const { onPageChange, data, currentPage, totalCount, pageSize, setPageSize } =
+    props;
 
   return (
-    <>
-      {data.map((recipe) => {
-        return <Recipe data={recipe} key={recipe._id} />;
-      })}
-      <Pagination
-        currentPage={currentPage}
-        totalCount={totalCount}
-        pageSize={pageSize}
-        onPageChange={(page) => onPageChange(page)}
-      />
-    </>
+    <div className="container">
+      <div className="grid-container">
+        {data.map((recipe) => {
+          return <Recipe data={recipe} key={recipe._id} />;
+        })}
+      </div>
+      <div className="d-flex justify-content-around mt-2">
+        <Pagination
+          currentPage={currentPage}
+          totalCount={totalCount}
+          pageSize={pageSize}
+          onPageChange={(page) => onPageChange(page)}
+        />
+        <div>
+          <label style={{ marginRight: '5px' }}>Recipes per page:</label>
+          <select onChange={(e) => setPageSize(e.target.value)}>
+            <option>5</option>
```

Comment:
you might want to start with something like 25, rather than have so many options. Also, usually a toggle type is better than a dropdown

Reply:
removed the first two options from dropdown. Kept it as dropdown